fragment.go

```diff
@@ -293,11 +293,13 @@ func (f *Fragment) close() error {
 	// Flush cache if closing gracefully.
 	if err := f.flushCache(); err != nil {
 		f.logger().Printf("fragment: error flushing cache on close: err=%s, path=%s", err, f.path)
+		return err
```

Comment:
I agree that we should handle them. I don't remember why they were previously ignored. It doesn't make a lot of difference in server shutdown but it can be helpful if we're spinning up multiple instances during tests or if someone embeds pilosa inside their application for some reason.